mobile/docs/root/api/starting_envoy.rst

```diff
@@ -545,6 +545,22 @@ A maximum of 100 entries will be stored.
   builder.enableDNSCache(true, saveInterval: 60)
 
 
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+``setRuntimeGuard``
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Adds a runtime guard key value pair to envoy configuration.  The guard is of the short form "feature"
+rather than the fully qualified "envoy.reloadable_features.feature"
+Note that Envoy will fail to start up in debug mode if an unknown guard is specified.
+
+**Example**::
+
+  // Kotlin
+  builder.setRuntimeGuard("feature", "true")
+
+  // Swift
+  builder.setRuntimeGuard("feature", "true")
```

Comment:
Can we please expose typesafe APIs to prevent invalid values from being passed from consumers and clarify the expected input?

Even if we stringify it to pass it through various layers internally, I'd rather have a safer and clearer public API.